portfolio/src/main/java/com/google/sps/data/BlogHashMap.java

```diff
@@ -24,7 +24,8 @@
 import java.util.Set;
 
 public final class BlogHashMap {
-  private final String[] INTERNAL_TAGS = {"#general", "#animals", "#art", "#business", "#education", "#environment", "#games", "#music", "#politics", "#wellbeing", "Tags"};
+  private final String[] INTERNAL_TAGS = {"#general", "#animals", "#art", "#business", "#education", 
```

Comment:
Any reason we can't use the constant variables defined in InternalTags here? One of the keys of defining constant variables is for reuse.

Reply:
Please use the InternalTags class for all things internal tags

Reply:
Lines 27&28  have no effect on the rest of the code